packages/eslint-config-universe/shared/typescript.js

```diff
@@ -75,6 +75,12 @@ module.exports = {
           { vars: 'all', args: 'none', ignoreRestSiblings: true, caughtErrors: 'all' },
         ],
 
+        'no-unused-expressions': 'off',
+        '@typescript-eslint/no-unused-expressions': [
+          'warn',
+          { allowShortCircuit: true, enforceForJSX: true },
+        ],
```

Comment:
Keep the chunks alphabetically sorted (`@typescript-eslint/no-unused-expressions` above `@typescript-eslint/no-unused-vars`)